google-home-community.groovy

```diff
@@ -1904,9 +2157,57 @@ private executeCommand_ActivateScene(deviceInfo, command) {
     return [[:], [:]]
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private executeCommand_ArmDisarm(deviceInfo, command) {
+    def armDisarmTrait = deviceInfo.deviceType.traits.ArmDisarm
+    def checkValue = "${armDisarmTrait.armValues["disarmed"]}"
+    def codePosition
```

Comment:
This doesn't need to be declared outside the `if` block.

Reply:
They are used inside the if and the else though?

Reply:
Sure, but they're never used outside the scope where they're set.  Just declare this variable in each branch.

Reply:
I see what you mean.  Moved codePosition inside each block.  checkValue needs a default that isn't set in 2/3 of those if-else blocks.